src/main/resources/templates/index.twig

```diff
@@ -11,4 +11,11 @@
     </li>
     {% endfor %}
 </ul>
-{% endblock %}
\ No newline at end of file
+<form method="POST">
```

Comment:
Also I'd use `post` instead of `POST` ([source](http://stackoverflow.com/a/4106613/3185307))